common/lib/capa/capa/inputtypes.py

```diff
@@ -127,6 +127,9 @@ def __repr__(self):
     def __eq__(self, other):
         return self._status == str(other)
 
+    def __hash__(self):
```

Comment:
@feanil  Py2 defines a hash value which is based on memory location. While in py3 that hash value is not available. So defined a hash method for class, this is not memory based so not comfortable with this but this does passes the test. I would be best if some one familiar with code could suggest a better hash method.

Reply:
I was going to say that we should use all the slots values, but it looks like they all derive from `self._status`, so it seems fine.